setup.py

```diff
@@ -218,6 +218,7 @@ def get_extensions():
         "nvcc": [
             "-O3" if not debug_mode else "-O0",
             "-t=0",
+             "-std=c++20"
```

Comment:
:fire:

Reply:
I don't think we can land this ahahh I like it but I'm sure somewhere there we don't support a new enough compiler

Reply:
In ao, you most likely can if you don't care about windows.
If you do care about windows, "most likely" -> "most likely can't" 
:D